projects/ccf-eui/src/app/core/core.module.ts

```diff
@@ -13,12 +14,17 @@ import { StoreModule } from './store/store.module';
 @NgModule({
   imports: [
     HttpClientModule,
+
     AnalyticsModule.forRoot({
       gaToken: environment.googleAnalyticsToken,
       appName: 'eui'
     }),
     MousePositionTrackerModule,
 
+    CcfApiModule.forRoot(() => new CcfApiConfiguration({
+      basePath: environment.dbOptions.remoteApiEndpoint
```

Comment:
How is this supposed to work with the webcomponent's remote-api input? Or is that still a work in progress?

Reply:
The path is set right before doing the request. This is just the initial value. See: https://github.com/hubmapconsortium/ccf-ui/pull/779/files#diff-a1ceb8e958659afbe8d2580371ced8535cea1197394692ab59cb6f2dbcc75addR188